src/AppInstallerCLICore/Command.cpp

```diff
@@ -933,7 +934,8 @@ namespace AppInstaller::CLI
     {
         try
         {
-            if (!Settings::User().GetWarnings().empty())
+            if (!Settings::User().GetWarnings().empty() &&
+                !WI_IsFlagSet(command->GetOutputFlags(), CommandOutputFlags::IgnoreSettingsWarnings))

```

Comment:
As discussed, I would think this would be better done using the output channels mechanism, but this is acceptable.

The nice thing about that mechanism being built out is that any error/exception could be output in a JSON format as well.  But we have only so much time.